### ISSUE TITLE
attach second network to a container after is created with chatflow 

### ISSUE THREAD

**Issue by Pishoy** [threefoldtech/jumpscaleX_threebot#772]
is valid to add an option in chatflow for attaching an existing container to a created network in chatflow , it will be useful if someone deletes the default created network for a container ?

**Comment by grimpy** [comment]
@zaibon is this possible if you send a new reservation of the same container? Or it's not possible in zos atm?

**Comment by zaibon** [comment]
No this is not possible and not on the roadmap. Container are static things define once. If you need to modify it, cancel and re-create.

**Comment by grimpy** [comment]
@zaibon but I guess because of the way the bridge names on zos are made (based on the name of the network) it would always be possible to 'recreate' the network with the same information right?

**Comment by zaibon** [comment]
I don't think this works... I believe we try to see if the network would come back to live but I doubt it. It's worth trying it out though.

**Comment by grimpy** [comment]
@Pishoy I suggest you try it out and let us know
